src/debug/dotNetDebugProvider.ts

```diff
@@ -46,6 +47,16 @@ export class DotNetDebugProvider implements IDebugProvider {
                 quoteArgs: false
             }
         };
+        const map = extensionConfig.getDotnetDebugSourceFileMap();
+        if (map) {
+            try{
```

Comment:
Nitpick: space before the brace
```suggestion
            try {
```